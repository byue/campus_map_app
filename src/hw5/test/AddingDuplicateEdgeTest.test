# Tests adding duplicate edge to graph.

# Create a graph
CreateGraph graph1

# Add 2 nodes
AddNode graph1 n1
AddNode graph1 n2

# add same edge twice
AddEdge graph1 n1 n2 a
AddEdge graph1 n1 n2 a

# add a new edge with node arguments reversed
AddEdge graph1 n2 n1 a