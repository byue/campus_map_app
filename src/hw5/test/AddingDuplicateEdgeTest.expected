# Tests adding duplicate edge to graph.

# Create a graph
created graph graph1

# Add 2 nodes
added node n1 to graph1
added node n2 to graph1

# add same edge twice
added edge a from n1 to n2 in graph1
graph1 already contains the edge

# add a new edge with node arguments reversed
added edge a from n2 to n1 in graph1